features/click-to-load.json

```diff
@@ -598,6 +598,9 @@
         "domain": "freenom.com",
         "reason": "https://github.com/duckduckgo/privacy-configuration/issues/1109"
     }, {
+        "domain": "isra.org",
+        "reason": "https://github.com/duckduckgo/privacy-configuration/issues/1245"
+
```

Comment:
@icodebyamanda I think your missing something here. Probably need to add some brackets.

Reply:
@SlayterDev thanks. Just added the brackets.